ios/hdfs/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you do contrib stuff as a separate PR? I suspect we're going to need to do a lot of work to make tests pass, etc.